cranelift-wasm/Cargo.toml

```diff
@@ -26,8 +26,8 @@ target-lexicon = "0.9"
 
 [features]
 default = ["std", "basic-blocks"]
-std = ["cranelift-codegen/std", "cranelift-frontend/std", "wasmparser/std"]
-core = ["hashbrown", "cranelift-codegen/core", "cranelift-frontend/core", "wasmparser/core"]
+std = ["cranelift-codegen/std", "cranelift-frontend/std"]
+core = ["hashbrown", "cranelift-codegen/core", "cranelift-frontend/core"]
```

Comment:
I wasn't sure about removing these but it seemed that wasmparser no longer had these features?

Reply:
lgtm